frontend/pages/hosts/HostDetailsPage/_styles.scss

```diff
@@ -679,12 +681,29 @@
         // Width for all columns except the "Query name" column
         // Width calculation adjusts for each row's horizontal padding
         th {
-          width: calc(200px - 27px * 2);
-        }
+          // Width for the "Query name" column
+          &:first-child {
+            width: 40%;
+            min-width: 100px;
+            padding-right: 0px;
+          }
+
+          &:nth-child(2) {
+            width: 12%;
+            min-width: 70px;
+          }
 
-        // Width for the "Query name" column
-        th:nth-child(1) {
-          width: auto !important;
+          &:nth-child(3) {
+            width: 18%;
+            min-width: 90px;
+          }
+
+          &:last-child {
+            width: 30%;
+            min-width: 100px;
+            max-width: 140px;
+            padding-right: 0px;
+          }
```

Comment:
From my understanding of React Table, if the `width` of a cell is set to a percentage, it disregards `min-width` and `max-width` properties altogether. It will always scale to whatever the percentage is set to. Would you please confirm that is the case, and if so remove the `min-width` and `max-width` properties?